examples/chaiscript/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ChaiScript)
+
+# DOCUMENTATION_START {
+hunter_add_package(chaiscript)
+find_package(chaiscript CONFIG REQUIRED)
+# DOCUMENTATION_END }
+
+if (CMAKE_HOST_UNIX) 
+    #set unix link libraries
+    list(APPEND LIBS "dl") 
+    list(APPEND LIBS "pthread")
+#    list(APPEND LIBS "readline")
+    add_definitions(-pthread)
```

Comment:
Use `Threads::Threads` instead:
* https://cmake.org/cmake/help/latest/module/FindThreads.html

Reply:
The example uses: 
target_link_libraries(chaitest ${LIBS} chaiscript::parser chaiscript::stdlib) 

chaiscript::parser and chaiscript::stdlib are static libraries;  I'm uncertain how to apply Threads::Threads in this case, short of possibly converting this to a shared library instead?

Reply:
You should use
```cmake
find_package(Threads REQUIRED)
target_link_libraries(... PUBLIC Threads::Threads)
```
in chaiscript sources and add
```cmake
find_package(Threads REQUIRED)
```
to generated `chaiscriptConfig.cmake`